src/main/java/com/jcabi/http/request/FakeRequest.java

```diff
@@ -116,8 +118,7 @@ public Response send(final Request req, final String home,
     /**
      * Content received.
      */
-    @Immutable.Array
-    private final transient byte[] content;
+    private final transient Map<Pattern, byte[]> content;
```

Comment:
@dmzaytsev probably - just thinking of the concept here but you should check jcabi-immutable. The name looks correct :)